articles/active-directory/devices/concept-primary-refresh-token.md

```diff
@@ -164,6 +164,9 @@ The following diagrams illustrate the underlying details in issuing, renewing, a
 | F | Azure AD validates the Session key signature by comparing it against the Session key embedded in the PRT, validates the nonce and verifies that the device is valid in the tenant and issues a new PRT. As seen before, the PRT is again accompanied with the Session key encrypted by Transport key (tkpub). |
 | G | CloudAP plugin passes the encrypted PRT and Session key to CloudAP. CloudAP requests the TPM to decrypt the Session key using the Transport key (tkpriv) and re-encrypt it using the TPM’s own key. CloudAP stores the encrypted Session key in its cache along with the PRT. |
 
+> [!NOTE]
+> PRT can be renewed externally without the need of a VPN connection when usernamemixed endpoints only are enabled externally.
```

Comment:
Remove "only" after usernamemixed endpoints. It's confusing and does not convey the point correctly. The requirement for usernamemixed was already specified above